tensorboard/webapp/runs/store/utils_test.ts

```diff
@@ -90,5 +90,141 @@ describe('run store utils test', () => {
         });
       });
     });
+
+    describe('by regex', () => {
+      it('do not change the group when the regex is empty', () => {
```

Comment:
`does not group when the regex is empty` (this method does not really change anything)